modules/ROOT/pages/deploy-to-runtime-fabric.adoc

```diff
@@ -32,6 +32,9 @@ Before deploying an application to Anypoint Runtime Fabric, perform the followin
 * Check if you require the *Exchange Contributors* permission to deploy applications to Runtime Fabric.
 * To deploy an application to a specific domain (endpoint), ensure that you have set up domains (endpoints) in your environment. Refer to xref:custom-ingress-configuration.adoc[Configure Ingress For Runtime Fabric].
 
+[IMPORTANT]
```

Comment:
Isn't it better if this is just presented as an additional bulleted item? It is also a prerequisite right?

Reply:
Yeah, I think you're right, but I'm just worried people will miss this, and won't be able to deploy.